src/views/signedExtrinsics/index.tsx

```diff
@@ -140,21 +139,22 @@ const SignedExtrinsics = () => {
   return (
     <div className="disable-vertical-scroll grid h-full grid-rows-[40px_46px_1fr] gap-8">
       <PageHeader title="Extrinsics" />
-      <SearchBar
-        label="Search by Block"
-        type="extrinsics"
-      />
-
       {
-        isLoading
-          ? <Loader />
-          : (
-            <TableComponent
-              columns={columns}
-              data={signedExtrinsics}
-              onRowClick={handleOpenModal}
-            />
+        !isLoading
+          ? (
+            <>
+              <SearchBar
+                label="Search by Block"
+                type="extrinsics"
+              />
+              <TableComponent
+                columns={columns}
+                data={signedExtrinsics}
+                onRowClick={handleOpenModal}
+              />
+            </>
           )
+          : <Loader />
```

Comment:
please use positive condition if possible ... is easy to read ... isLoading ? loading : no loading
check the deleted part

Reply:
Done